wording enhancement: CDS Service is subject to user authz (#617)
cds-hooks/docs: docs/specification/current.md

- The CDS Client MAY compute an efficient set of prefetch templates from multiple CDS Services, thereby reducing the number of calls to a minimum
- The CDS Client MAY satisfy some of the desired prefetched templates via some internal service or even its own FHIR server.

> The CDS Client SHALL only provide access to the requested resource if it is within the user's authorized scope.

Reviewer: I'd rather say...

```suggestion
The CDS Client SHALL only provide access to the resources that the user is authorized to see
```

... but the underlying Jira issue specifies this language. (In which case, why review? ;-))
